developer/utils/caching.py

```diff
@@ -46,15 +46,9 @@ def _cache_descriptor(sg, desc_type, desc_dict, target_path):
 
 def _should_skip_caching(desc):
     """
-    Returns if a descriptor's content should not be cached.
-
-    We should not attempt to cache descriptors that are path-based. Not only they don't
-    need to be cached, but they might be using special tokens like CONFIG_FOLDER
-    that can't be understood outside a pipeline configuration.
-
-    :returns: ``True`` if the contents should be skipped, ``False`` otherwise.
+    By default, everything is cached.
     """
-    return desc["type"] in ["dev", "path"]
+    return False
```

Comment:
Why do we have this at all since the value is fixed?

Reply:
Ah I see you are passing it as a callable. I would probably have used a lambda, but 🤷‍♂ doesn't matter too much either way.